public/app/features/alerting/unified/rule-list/filter/types.ts

```diff
@@ -14,4 +14,5 @@ export type AdvancedFilters = {
   dashboardUid?: string;
   plugins?: 'show' | 'hide';
   contactPoint?: string | null;
+  ruleSource?: '*' | RuleSource;
```

Comment:
Just a thought - isn't easier to treat `undefined` as '*'? Currently we need to support both `undefined` and `*`